src/rleCompress.js

```diff
@@ -19,6 +19,35 @@
  */
 function rleCompress(source) {
   // write code here
+  let compressedString = '';
+  let i = 0;
+  let amountFollowingLetters = 0;
+
+  while (i < source.length) {
+    let currentLetter = source[i];
+
+    if (currentLetter !== compressedString[compressedString.length - 1]) {
+      // I think, that's not bad to write following code, but what is in practice?
+      compressedString += ((amountFollowingLetters > 1) ? amountFollowingLetters : '') + currentLetter;
```

Comment:
ternary is ok, but ` + currentLetter;` with ternary is too complex
use if/else like below or transform it to ternary